### PR TITLE
`[Exiled::Events]` Added new commands for installing plugins, changing or printing config values

### PR DESCRIPTION
**Pull request by VALERA771**
`[Exiled::API]` Added extension 
```csharp
StringExtensions::FromSnakeCase(string)
```
for converting strings from snake_case to NotSnakeCase

### CODE REVIEW

**Comment by VALERA771** [comment]
Install command tested. Other will be tomorrow

File: Exiled.Events/Commands/ConfigValue/Set.cs
Comment (louis1706) [inline Exiled.Events/Commands/ConfigValue/Set.cs:82]: Add a try catch in case the value give is not correct

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.Events/Commands/ConfigValue/Set.cs
Comment (VALERA771) [reply Exiled.Events/Commands/ConfigValue/Set.cs:82]: I think it'll return null instead of throwing. I'll check that

Comment (louis1706) [reply Exiled.Events/Commands/ConfigValue/Set.cs:82]: > I think it'll return null instead of throwing. I'll check that

It's throwing if invalid

**Comment by VALERA771** [comment]
All commands have been tested

**Review by louis1706 — APPROVED** [review]

**Review by Vladislav-CS — APPROVED** [review]

**Review by Misfiy — APPROVED** [review]

**Comment by Vladislav-CS** [comment]
Why DoNotMerge?

**Comment by Misfiy** [comment]
@NaoUnderscore (as asked by vlad) why do not merge?

**Comment by NaoUnderscore** [comment]
Because I'd like to wait before merging Plugin Manager/Installer-like features.
We're going to implement it through our Web APIs.

**Comment by VALERA771** [comment]
@NaoUnderscore

**Pull request merged**
